audit-service/src/contract-tests/java/uk/gov/gchq/palisade/contract/audit/rest/HealthActuatorContractTest.java

```diff
@@ -23,50 +23,48 @@
 import org.springframework.boot.test.web.client.TestRestTemplate;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
+import org.springframework.test.context.ActiveProfiles;
 
 import uk.gov.gchq.palisade.service.audit.AuditApplication;
-import uk.gov.gchq.palisade.service.audit.request.AuditRequest;
 import uk.gov.gchq.palisade.service.audit.service.AuditService;
 
-import java.util.List;
 import java.util.Map;
 
 import static org.assertj.core.api.Assertions.assertThat;
 
-
-@SpringBootTest(classes = AuditApplication.class, webEnvironment = WebEnvironment.RANDOM_PORT)
- class RestControllerContractTest extends AuditTestCommon {
+/**
+ * An external requirement of the service is to keep-alive in k8s.
+ * This is done by checking the service is still alive and healthy by REST GET /actuator/health.
+ * This should return 200 OK if the service is healthy.
+ */
+@SpringBootTest(
+        classes = AuditApplication.class,
+        webEnvironment = WebEnvironment.RANDOM_PORT,
+        properties = {"spring.autoconfigure.exclude=org.springframework.boot.autoconfigure.data.redis.RedisAutoConfiguration"}
+)
+@ActiveProfiles({"akkatest"})
+ class HealthActuatorContractTest {
 
     @Autowired
     private TestRestTemplate restTemplate;
     @Autowired
     private Map<String, AuditService> serviceMap;
 
-    List<AuditRequest> requests = List.of(
-            readRequestCompleteAuditRequest(),
-            readRequestExceptionAuditRequest(),
-            registerRequestCompleteAuditRequest(),
-            registerRequestExceptionAuditRequest()
-    );
-
     @Test
      void testContextLoads() {
-        assertThat(serviceMap)
-                .isNotNull()
+        assertThat(serviceMap).isNotNull()
                 .isNotEmpty();
+        assertThat(restTemplate).isNotNull();
     }
 
     @Test
      void testIsUp() {
```

Comment:
More of a question.  We can get an OK status for a HTTP response for any number of reasons.  I can also see that sometimes we only go for the status and other times, we also check to see if the body contains the string "status: UP".  What is the reason for this?

Reply:
Will look into this. There are a different number of asserts that are done for the health actuator for the services. Probably best to check both the Http code and the response body.

Reply:
I've added an assertion to check the body of the response as well as the returned http code.